Dockerfile-alpine.template

```diff
@@ -77,17 +77,12 @@ RUN set -ex; \
 
 COPY php.ini /usr/local/etc/php/conf.d/php-matomo.ini
 
-# https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/
-#RUN set -ex; \
-#	curl -fsSL -o GeoIPCity.tar.gz \
-#		"https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz"; \
-#	curl -fsSL -o GeoIPCity.tar.gz.md5 \
-#		"https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz.md5"; \
-#	echo "$(cat GeoIPCity.tar.gz.md5)  GeoIPCity.tar.gz" | md5sum -c -; \
-#	mkdir /usr/src/GeoIPCity; \
-#	tar -xf GeoIPCity.tar.gz -C /usr/src/GeoIPCity --strip-components=1; \
-#	mv /usr/src/GeoIPCity/GeoLite2-City.mmdb /usr/src/matomo/misc/GeoLite2-City.mmdb; \
-#	rm -rf GeoIPCity*
+RUN \
+  set -ex; \
+  curl -fsSL -o DBIP-City.mmdb.gz "https://download.db-ip.com/free/dbip-city-lite-2020-02.mmdb.gz"; \
```

Comment:
As matomo is in the official library this is not great. Some verification would be great. For some examples please look at https://github.com/docker-library/official-images#image-build

Reply:
I'd prefer the matomo docker images not to be bloated by 80 MB (current size is 145 MB).
I have the dbip on the host and map it as a volume.